src/Components/Accordion.js

```diff
@@ -0,0 +1,61 @@
+import React,{useState} from 'react'
+import arrow from '../images/icon-arrow-down.svg';
+import styled,{css} from 'styled-components'
+const Wrapper = styled.div`
```

Comment:
I'd suggest moving all styles to separate file called `styles.js` located in the same folder exporting set of styled components.
Later you can import your components:
```
// styles.js
import styled from 'styled-components';

export const Wrapper = styled.div`
  // some styles
`


// Accordion.js
import {Wrapper} from ./styles

<Wrapper>...</Wrapper>
```